strawberryfields/configuration.py

```diff
@@ -218,6 +220,49 @@ def parse_environment_variable(key, value):
 
     return value
 
+def store_account(authentication_token, filename="config.toml", location="user_config", **kwargs):
+    """Stores an account in a configuration file.
+
+    The configuration file can be created in the following locations:
+       - user configuration directory ("user_config")
+       - current working directory ("local")
+
+    Args:
+        authentication_token (str): the authentication token to use when
+            connecting to the API; it will be sent with every request in the
+            header
```

Comment:
Since this is user facing, we can make it less technical. How about something like the following?
```suggestion
        authentication_token (str): API token for authentication to the Xanadu Cloud platform.
            This is required for submitting remote jobs using :class:`~.StarshipEngine`.
```

Reply:
Sounds good! I'll update it in `configuration.rst` as well (got it from there)